en/index.md

```diff
@@ -524,6 +524,46 @@ For example:
 
 - **Don’t use:** Click the ![pencil](https://assets.ubuntu.com/v1/633aab2b-pencil.png) pencil icon to edit the post.
 
+## Hyperlinks
+
+Here are some pointers about the general usage of hyperlinks and how to format them correctly.
+
+### General usage
+Avoid excessive links in the same paragraph or instruction. If you find yourself introducing several links in your content, consider listing them in a separate section called "Related topics", "Additional resources", or similar.
+
+When possible, avoid external links. They can easily become outdated, making the document harder to maintain.
+
+When linking to versioned files or specific lines of code, copy the **permalink** instead of the URL. This will ensure the link is bound to the current revision of the file, so it will direct to the same content even if the file changes.
+
+### Formatting
+Make the link text match the exact title or heading that you are referencing. Avoid phrases like "this document", "this article", "click here".
+
+If linking to a section called "Formatting":
+
+- Use: `[Formatting](#formatting)` 
+- Don't use: `[Guidelines to format hyperlinks](#formatting)`
+- Don't use: `[this section](#formatting)`
+
+In a full sentence, introduce the link with "For more information, see ..." or similar. 
+
+- Use: `For more information about formatting hyperlinks, see [Formatting](#formatting)`
+- Don't use: `See [Formatting](#formatting).`
+
+Avoid using a URL as the linked text.
+- Use: `[Page title](https://page-url.com)`
+- Don't use: `[https://page-url.com](https://page-url.com)`
+
+For external links, inform the user the link is external to the docs by specifying the source. This can either be done as part of the sentence, or with a delimiter.
+
+- Use: `For more information, see the ExternalStyleGuide page [How to format hyperlinks](www.external-style-guide.com/hyperlinks).`
+- Use: `- ExternalStyleGuide | How to format hyperlinks](www.external-style-guide.com/hyperlinks)`
+- Don't use: `For more information, see [How to format hyperlinks](www.external-style-guide.com/hyperlinks)`
+
+If the link performs an action, like downloading a file, link the entire action in the sentence. 
+
+- Use: `First, [download the file](file.zip)`
```

Comment:
- I'd suggest including the extension in the link text, too (file.zip), so the user knows what to expect.
- Given our guidance regarding [Inline formatting](https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/style-guide/#inline-formatting), I'd use monospace for the file:
   ```
   [download the `file.zip`](file.zip)
   ```